pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -0,0 +1,252 @@
+          description: this.t('epinio.intro.cards.namespaces.description'),
+          slotTitle:   this.t('epinio.intro.cards.namespaces.slotTitle')
+        },
+        {
+          title:       this.t('typeLabel.applications', { count: this.namespaces?.totalApps }),
+          icon:        'icon-application',
+          cta:         createEpinioRoute('c-cluster-applications-createapp', { resource: EPINIO_TYPES.APP }),
+          link:        createEpinioRoute('c-cluster-applications', { resource: EPINIO_TYPES.APP }),
+          linkText:    this.t('epinio.intro.cards.applications.linkText'),
+          description: this.t('epinio.intro.cards.applications.description'),
+          slotTitle:   this.t('epinio.intro.cards.applications.slotTitle')
+        },
+        {
+          title:       this.t('epinio.intro.cards.services.title'),
+          icon:        'icon-service',
+          cta:         createEpinioRoute('c-cluster-resource-create', { resource: EPINIO_TYPES.SERVICE_INSTANCE }),
+          link:        createEpinioRoute('c-cluster-resource', { resource: EPINIO_TYPES.SERVICE_INSTANCE }),
+          linkText:    this.t('epinio.intro.cards.services.linkText'),
+          description: this.t('epinio.intro.cards.services.description'),
+          slotTitle:   this.t('epinio.intro.cards.services.slotTitle')
+        }],
+      colorStops: {
+        0: '--info', 30: '--info', 70: '--info'
+      },
+    };
+  },
+  computed: {
+    servicesStarter() {
+      const fetchServices = this.$store.getters['epinio/all'](EPINIO_TYPES.CATALOG_SERVICE);
+      const services: EpinioCatalogService[] = fetchServices.length ? fetchServices.slice(0, 2) : [{ id: 'redis-dev-t' }, { id: 'mysql-dev' }];
+
+      const s = services.reduce((acc: any[], service) => {
+        acc.push({
+          link: createEpinioRoute('c-cluster-resource-create', { resource: EPINIO_TYPES.SERVICE_INSTANCE, name: service?.shortId }, { query: { service: service?.shortId } }),
+          id:   service?.shortId
+        });
+
+        return acc;
+      }, []);
+
+      return s;
+    },
+    version() {
+      const { displayVersion } = getVersionInfo(this.$store);
+
+      return displayVersion;
+    },
+    apps() {
+      const allApps = this.$store.getters['epinio/all'](EPINIO_TYPES.APP);
+
+      const runningApps = allApps.filter((app: EpinioApplicationResource) => app.status === 'running' );
+      const stoppedApps: EpinioApplicationResource[] = allApps.filter((app: EpinioApplicationResource) => app.status === 'error' );
+
+      return {
+        runningApps: runningApps.length,
+        stoppedApps: stoppedApps.length,
+        totalApps:   allApps.length
+      };
+    },
+    namespaces() {
+      const allNamespaces = this.$store.getters['epinio/all'](EPINIO_TYPES.NAMESPACE);
+
+      let totalApps = 0;
+      let totalConfigurations = 0;
+
+      return allNamespaces.reduce((acc: any, namespace: any) => {
```

Comment:
Total apps and configurations can just come from the apps and configurations lists, rather than summing from namespaces